dependencies/direct/py-constraints.in

```diff
@@ -13,3 +13,5 @@
 #      remain in Git.                                                         #
 #                                                                             #
 ###############################################################################
+
+awx_plugins.interfaces @ git+https://github.com/webknjaz/ansible--awx_plugins.interfaces.git@feature/shared-awx-stuff
```

Comment:
This PR is kept in draft until [0a638cd](https://github.com/ansible/awx-plugins/pull/38/commits/0a638cdd0f275f71683d5b53d3bc3634ae1e5bf8) is reverted.